File: redwood-yoga-java/src/main/java/io/github/orioncraftmc/meditate/internal/enums/YGAlign.java
package io.github.orioncraftmc.meditate.internal.enums;

public enum YGAlign //Type originates from: YGEnums.h
{
    YGAlignAuto,
    YGAlignFlexStart,
    YGAlignCenter,
    YGAlignFlexEnd,
    YGAlignStretch,
    YGAlignBaseline,
    YGAlignSpaceBetween,
    YGAlignSpaceAround;

    public static final int SIZE = java.lang.Integer.SIZE;

    public static YGAlign forValue(int value) {
        return values()[value];
    }

    public int getValue() {
        return this.ordinal();
    }
}

File: redwood-yoga/src/commonMain/kotlin/app/cash/redwood/yoga/internal/enums/YGAlign.kt
package app.cash.redwood.yoga.internal.enums

/** Type originates from: YGEnums.h */
enum class YGAlign {
  YGAlignAuto,
  YGAlignFlexStart,
  YGAlignCenter,
  YGAlignFlexEnd,
  YGAlignStretch,
  YGAlignBaseline,
  YGAlignSpaceBetween,
  YGAlignSpaceAround;

  fun getValue(): Int {
    return ordinal
  }

  companion object {
    fun forValue(value: Int): YGAlign {
      return values()[value]
    }
  }
}
